docs/index.md

```diff
@@ -88,6 +88,10 @@ export SPECTROCLOUD_APIKEY=5b7aad.........
 provider "spectrocloud" {}
 ```
 
+## Import
```

Comment:
@nikchern I think for this section, you don't need to list out resources. I think we're better off just explaining import and how it can be done through one or two examples. Maybe a context example for `tenant`, and `project` and one without context?  And then wrap it up by stating, each resource will have its own example.

Reply:
@karl-cardenas-coding thanks a ton for review, I put as discussed 3 examples here with reference to particular spectrocloud resources.